mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -396,7 +399,18 @@ def _should_parse_as_unified_llm_input(json_input: dict):
     return any(x in json_input for x in SUPPORTED_LLM_FORMATS)
 
 
+class ParsedJsonInput(NamedTuple):
+    data: Any
+    params: Optional[Dict]
+    should_parse_as_unified_llm_input: bool
+
```

Comment:
```suggestion
    is_unified_llm_input: bool
```
nit: because it's already parsed?